cmds/shutdown/shutdown.go

```diff
@@ -34,11 +45,18 @@ var (
 )
 
 func usage() {
-	log.Fatalf("shutdown [-dryrun] [halt|reboot|suspend] (defaults to reboot)")
+	fmt.Printf("shutdown [-dryrun] [-k] [-h|-r|halt|reboot|suspend] (defaults to reboot)")
+	os.Exit(1)
 }
 
-func main() {
-	flag.Parse()
+func init() {
+	flag.BoolVar(&flags.dryrun, "dryrun", false, "Do not do kexec system calls")
+	flag.BoolVar(&flags.k, "k", false, "Do not do kexec system calls")
```

Comment:
k and dryrun are the same thing can you just have them point to the same variable? What's different?

Reply:
I didn't realise that was possible.  That would make it much simpler.